### PR TITLE
11 09 footer

### PR DESCRIPTION
**Pull request by shikigami12**
Added styles for footer (mobile|tablet|desktop)
Styles:
Desktop actual:
image
Desktop expected:
image
Tablet actual:
image
Tablet expected:
image
Mobile actual:
image
Mobile expected:
image

### CODE REVIEW

**Review by PavloRohozhyn — APPROVED** [review]
well done

File: src/css/layout/09-footer.css [hunk 1]
```diff
@@ -1,2 +1,226 @@
...
+    color: rgba(255, 255, 255, 0.60);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.28;
+    letter-spacing: -0.28px;
+}
+
+.footer__address--phone--link,
+.footer__address--address--text--second-line {
```
Comment (krav-yurii) [inline src/css/layout/09-footer.css:88]: This line can be deleted

File: src/css/layout/09-footer.css [hunk 2]
```diff
@@ -1,2 +1,226 @@
...
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.28;
+}
+
+.footer__links--link:hover {
+    color: #4caf50;
+}
+
```
Comment (krav-yurii) [inline src/css/layout/09-footer.css]: I think we should use the same styles in terms of animations for these buttons, both for the header and for the footer

Comment (shikigami12) [reply src/css/layout/09-footer.css]: fixed

File: src/css/layout/09-footer.css (hunk 1 above)
Comment (shikigami12) [reply src/css/layout/09-footer.css:88]: fixed

**Review by Westflag — APPROVED** [review]

**Review by PavloRohozhyn — APPROVED** [review]

**Pull request merged**